x-pack/plugins/synthetics/public/apps/synthetics/components/common/screenshot/journey_step_screenshot_container.tsx

```diff
@@ -39,7 +39,9 @@ export const JourneyStepScreenshotContainer = ({
 
   const { basePath } = useContext(SyntheticsSettingsContext);
 
-  const imgPath = `${basePath}/internal/uptime/journey/screenshot/${checkGroup}/${initialStepNumber}`;
+  const imgPath = checkGroup
+    ? `${basePath}/internal/uptime/journey/screenshot/${checkGroup}/${initialStepNumber}`
+    : '';
```

Comment:
i actually wanted to do it as well, unfortunately underlying code is too complex. Since it will meant calling some conditional hooks. let me take a look once again if it's possible. Otherwise i will leave it as is for now.